javascript/packages/orchestrator/src/chainSpec.ts

```diff
@@ -96,6 +96,18 @@ export async function addBalances(specPath: string, nodes: Node[]) {
   try {
     const chainSpec = readAndParseChainSpec(specPath);
     const runtimeConfig = getRuntimeConfig(chainSpec);
+    // Create a balance map
+    const balanceMap = runtimeConfig.balances.balances.reduce(
```

Comment:
This ```balances.balances``` is strange to me, maybe we could find a better naming ? :thinking:

Reply:
This is not something we chose to name. Its coming from [Polkadot chainspec](https://github.com/paritytech/polkadot/blob/f963646940e3c38e359d1ecf76478d1fed36021c/node/service/src/chain_spec.rs#L324-L325)

Reply:
Exactly, the naming here is from the chain-spec.